scripts/build-agent

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Why are all these changes in this patch? Rebase issue?

Reply:
yes , I removed it.